routes/product-item.html

```diff
@@ -0,0 +1,219 @@
+                    </a>
+                </li>
+            </ul>
+        </nav>
+
+        <!-- Nav main ends -->
+        
+        <!-- Product Item Body -->
+
+        <div class="product-item-body">
+            <div class="section-left">
+                <img src="../assets/carousel/jacket.svg" alt="product image" class="product-image"/>
+                <p><i class="fa-solid fa-magnifying-glass-plus"></i> Roll over image to zoom in</p>
+            </div>
+            <div class="section-right">
+                <p class="product-title">
+                    Padded Jacket
+                </p>
+
+                <p class="product-id">
+                    ID: ABC12334
+                </p>
+
+                <form class="rating-form">
+                    <div class="rating-container">
+                      <label>
+                        <input type="checkbox" class="rating" value="1" name="radio" checked/>
+                        <i class="fas fa-star rating-icn"></i>
+                      </label>
+                      <label>
+                        <input type="checkbox" class="rating" value="2" name="radio" checked/>
+                        <i class="fas fa-star rating-icn"></i>
+                      </label>
+                      <label>
+                        <input type="checkbox" class="rating" value="3" name="radio" checked/>
+                        <i class="fas fa-star rating-icn"></i>
+                      </label>
+                      <label>
+                        <input type="checkbox" class="rating" value="4" name="radio" checked/>
+                        <i class="fas fa-star rating-icn"></i>
+                      </label>
+                      <label>
+                        <input type="checkbox" class="rating" value="5" name="radio" />
+                        <i class="fas fa-star rating-icn"></i>
+                      </label>
+                      4/5
+                    </div>
+                </form>
+
+                <hr>
+
+                <p class="item-price">
+                    Rs 3499<sup>00</sup> <strike>6999/-</strike>
+                </p>
+                <p class="item-discount">
+                    -50% off!
+                </p>
+                
+                <div class="item-offer-container">
+                    <p class="item-payment-offers">
+                        <i class="fa-solid fa-credit-card item-payment-icn"></i>Exciting offers with credit cards.
+                    </p>
+    
+                    <p class="item-payment-offers">
+                        <i class="fa-brands fa-cc-visa item-payment-icn"></i>Zero Interest EMI on select cards
+                    </p>
+                </div>
+                
+                <p class="item-stock">
+                    In Stock!
+                </p>
+
+                <div class="product-button-container">
+                    <button class="prod-btn btn-cta-buy">Buy Now</button>
```

Comment:
You could add hover to these buttons as well to keep the hover styling consistent as you've added it for the goto button. The rest looks good!